open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/StyleFactory.java

```diff
@@ -71,19 +62,19 @@ public FeatureVisualizationStyle newStyle(Collection<? extends Action> actions,
     {
         FeatureVisualizationStyle style = null;
         FeatureVisualizationStyle defaultStyleCopy = clone(defaultStyle, dataType);
-        boolean containsLabelAction = false;
+//        boolean containsLabelAction = false;

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.